tests/api/activitypub/outbox.test.js

```diff
@@ -88,6 +88,15 @@ describe('outbox', () => {
       createActivity.object.attachment.should.be.an.Array()
     })
 
+    it('should return clean content when no item details', async () => {
+      const user = createUser({ fediversable: true })
+      const { username } = await user
+      await wait(debounceTime)
+      const outboxUrl = `${endpoint}${username}&offset=0`
+      const res = await publicReq('get', outboxUrl)
+      res.orderedItems[0].object.content.should.not.containEql('undefined')
```

Comment:
This test fails has `res.orderedItems` is an empty array: shouldn't there be a call to `createItem` somewhere?

Reply:
done in d7f5101c6